keycloak/deploy/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+KEYCLOAK_USER=admin
+KEYCLOAK_PASSWORD=ash12345^&*()
+
+if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
+    keycloak/bin/add-user-keycloak.sh --user $KEYCLOAK_USER --password $KEYCLOAK_PASSWORD
+fi
+
+
+export HOSTNAME_IP=$(hostname -i)
+export HOSTNAME_IP_ALL=$(hostname --all-ip-addresses)
+echo "hostname -i returned: $HOSTNAME_IP, -I returned: $HOSTNAME_IP_ALL"
+
+exec /opt/jboss/keycloak/bin/standalone.sh -Djboss.bind.address.private=$HOSTNAME_IP $@
```

Comment:
this seems to be conflicting syntax with the CMD provided in the dockerfile. 

I suggest that 
1. you move the computation of the $HOSTNAME_IP to the Dockerfile and 
2. Add -bprivate $HOSTNAME_IP as additional args in the CMD line.